test/integration/RemoteMicroservice.test.js

```diff
@@ -271,6 +278,12 @@ describe('The RemoteMicroservice Component', () => {
             expect(books).to.have.length(2);
         });
 
+        it.skip('does not expose models over the rest api', async function() {
```

Comment:
same here

Reply:
Same as above

Reply:
Nope it's not the same since public is not the default anymore! fixed it!